include/clap/entry.h

```diff
@@ -16,8 +16,8 @@ extern "C" {
 //   - /usr/lib/clap
 //
 // Windows
-//   - %CommonFilesFolder%/CLAP/
-//   - %LOCALAPPDATA%/Programs/Common/CLAP/
+//   - %COMMONPROGRAMFILES%\CLAP
+//   - %LOCALAPPDATA%\Programs\Common\CLAP
```

Comment:
Should this change be mentioned in the changelog? Since the earlier version was apparently incorrect.

Reply:
Yes, I'll do it.